plugin.video.infowars/default.py

```diff
@@ -354,65 +364,71 @@ def Menu_MainMenu(): #The Main Menu
 
 def Info_Wars_Live_Sub_Menu(title=''): #The Main Menu
     WhereAmI('@ Info Wars Live')
-    
+    http = urllib3.PoolManager()
     url = 'https://www.infowars.com/watch-alex-jones-show/'
-    response = urllib2.urlopen(url)
-    if response and response.getcode() == 200:
-        content = response.read()
+    response = http.request('GET', url)
+    if response:
+        content = response.data
         videos= find_multiple_matches(content,"<ul class=\"show__footer\"(.*?)</ul>")
         sources = []
         for temp, entry in enumerate(videos): 
             if temp == 0:
                 video_id = find_single_match(entry,"data-youtube-src=\"https://www.youtube.com/embed/(.*?)\"")
                 url = "plugin://plugin.video.youtube/play/?video_id=%s" % video_id
                 #IW_addon.log('SPINAL VIDEO_ID IS %s' % video_id)
-                PlayURL(url)
+                xbmc.Player().play(url)
                 break
     eod()
 
 def Real_News_WDK_Sub_Menu(title=''): #The Main Menu
     WhereAmI('@ Real News Live')
-    
+    http = urllib3.PoolManager()
     url = 'https://www.infowars.com/watch-alex-jones-show/'
-    response = urllib2.urlopen(url)
-    if response and response.getcode() == 200:
-        content = response.read()
+    response = http.request('GET', url)
+    if response:
+        content = response.data
         videos= find_multiple_matches(content,"<ul class=\"show__footer\"(.*?)</ul>")
         sources = []
         for temp, entry in enumerate(videos): 
             if temp == 1:
                 video_id = find_single_match(entry,"data-youtube-src=\"https://www.youtube.com/embed/(.*?)\"")
                 url = "plugin://plugin.video.youtube/play/?video_id=%s" % video_id
-                PlayURL(url)
+                play=xbmc.Player() ### xbmc.PLAYER_CORE_AUTO | xbmc.PLAYER_CORE_DVDPLAYER | xbmc.PLAYER_CORE_MPLAYER | xbmc.PLAYER_CORE_PAPLAYER
+                play.play(url)
                 break
     eod()
 
 def War_Room_Sub_Menu(title=''): #The Main Menu
     WhereAmI('@ War Room Live')
-    
+    http = urllib3.PoolManager()
     url = 'https://www.infowars.com/watch-alex-jones-show/'
-    response = urllib2.urlopen(url)
-    if response and response.getcode() == 200:
-        content = response.read()
+    response = http.request('GET', url)
+    if response:
+        content = response.data
         videos= find_multiple_matches(content,"<ul class=\"show__footer\"(.*?)</ul>")
         sources = []
         for temp, entry in enumerate(videos): 
             if temp == 2:
                 video_id = find_single_match(entry,"data-youtube-src=\"https://www.youtube.com/embed/(.*?)\"")
                 url = "plugin://plugin.video.youtube/play/?video_id=%s" % video_id
-                PlayURL(url)
+                play=xbmc.Player() ### xbmc.PLAYER_CORE_AUTO | xbmc.PLAYER_CORE_DVDPLAYER | xbmc.PLAYER_CORE_MPLAYER | xbmc.PLAYER_CORE_PAPLAYER

```

Comment:
same

Reply:
same

Lines 373 - 431 are commented out, old code, left in for reference for the other addon maintainer, but not actually executed.  I can delete that whole block and repull, but thought this would be ok.